app/Http/Controllers/Bible/TextController.php

```diff
@@ -202,27 +202,37 @@ public function search()
         })->flatten()->toArray();
 
         $search_text  = '%' . $query . '%';
+        $select_columns = [
+            'bible_verses.book_id as book_id',
+            'bible_books.bible_id as bible_id',
+            'books.name as book_name',
+            'bible_books.name as book_vernacular_name',
+            'bible_verses.chapter',
+            'bible_verses.verse_start',
+            'bible_verses.verse_end',
+            'bible_verses.verse_text',
+        ];
         $verses = BibleVerse::where('hash_id', $fileset->hash_id)
             ->withVernacularMetaData($bible)
             ->when($book_id, function ($query) use ($book_id) {
                 $books = explode(',', $book_id);
                 $query->whereIn('bible_verses.book_id', $books);
             })
-            ->where('bible_verses.verse_text', 'like', $search_text)
-            ->select([
-                'bible_verses.book_id as book_id',
-                'bible_books.bible_id as bible_id',
-                'books.name as book_name',
-                'bible_books.name as book_vernacular_name',
-                'bible_verses.chapter',
-                'bible_verses.verse_start',
-                'bible_verses.verse_end',
-                'bible_verses.verse_text',
-                'glyph_chapter.glyph as chapter_vernacular',
-                'glyph_start.glyph as verse_start_vernacular',
-                'glyph_end.glyph as verse_end_vernacular',
-            ]);
+            ->where('bible_verses.verse_text', 'like', $search_text);
```

Comment:
@bradflood just a question, I saw that verse_text column has a fulltext index, but we are not using it, maybe,  could we use the fulltext to do the search?